packages/jest-cli/src/cli/args.js

```diff
@@ -261,6 +261,12 @@ const options = {
       'Will run all tests affected by file changes in the last commit made.',
     type: 'boolean',
   },
+  forceExitAfterTestRun: {
```

Comment:
I'm not a fan of this name. Can we call it `forceExit` or `exitOnComplete` or something similar?